crates/wdl-engine/src/eval/v1/task.rs

```diff
@@ -1215,23 +1153,24 @@ impl TaskEvaluator {
         }
 
         // Write the outputs to the task's root directory
-        write_json_file(root_dir.join(OUTPUTS_FILE), &outputs)?;
+        write_json_file(task_eval_root.join(OUTPUTS_FILE), &outputs)?;
 
         evaluated.outputs = Ok(outputs);
         Ok(evaluated)
     }
+}
 
+impl<'a> State<'a> {
```

Comment:
I realize it's funky to break up the `impl State` blocks like this, but in addition to minimizing the diff in this PR I think it reads better from top-to-bottom this way.